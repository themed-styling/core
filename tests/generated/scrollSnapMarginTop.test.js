import 'jest-styled-components'

import { scrollSnapMarginTop } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(scrollSnapMarginTop, 'scrollSnapMarginTop')
