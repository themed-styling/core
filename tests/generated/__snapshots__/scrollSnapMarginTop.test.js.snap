// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`scrollSnapMarginTop renders 0 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-top: 0;
  -moz-scroll-snap-margin-top: 0;
  -ms-scroll-snap-margin-top: 0;
  scroll-snap-margin-top: 0;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: 0;
  -moz-scroll-snap-margin-top: 0;
  -ms-scroll-snap-margin-top: 0;
  scroll-snap-margin-top: 0;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: 0;
  -moz-scroll-snap-margin-top: 0;
  -ms-scroll-snap-margin-top: 0;
  scroll-snap-margin-top: 0;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: 0;
  -moz-scroll-snap-margin-top: 0;
  -ms-scroll-snap-margin-top: 0;
  scroll-snap-margin-top: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginTop renders function 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-top: hasnot;
  -moz-scroll-snap-margin-top: hasnot;
  -ms-scroll-snap-margin-top: hasnot;
  scroll-snap-margin-top: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: has;
  -moz-scroll-snap-margin-top: has;
  -ms-scroll-snap-margin-top: has;
  scroll-snap-margin-top: has;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: hasnot;
  -moz-scroll-snap-margin-top: hasnot;
  -ms-scroll-snap-margin-top: hasnot;
  scroll-snap-margin-top: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: has;
  -moz-scroll-snap-margin-top: has;
  -ms-scroll-snap-margin-top: has;
  scroll-snap-margin-top: has;
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginTop renders invalid theme path 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-top: invalid.path.1;
  -moz-scroll-snap-margin-top: invalid.path.1;
  -ms-scroll-snap-margin-top: invalid.path.1;
  scroll-snap-margin-top: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: invalid.path.1;
  -moz-scroll-snap-margin-top: invalid.path.1;
  -ms-scroll-snap-margin-top: invalid.path.1;
  scroll-snap-margin-top: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: invalid.path.1;
  -moz-scroll-snap-margin-top: invalid.path.1;
  -ms-scroll-snap-margin-top: invalid.path.1;
  scroll-snap-margin-top: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: invalid.path.1;
  -moz-scroll-snap-margin-top: invalid.path.1;
  -ms-scroll-snap-margin-top: invalid.path.1;
  scroll-snap-margin-top: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginTop renders negative number 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-top: -54881;
  -moz-scroll-snap-margin-top: -54881;
  -ms-scroll-snap-margin-top: -54881;
  scroll-snap-margin-top: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: -54881;
  -moz-scroll-snap-margin-top: -54881;
  -ms-scroll-snap-margin-top: -54881;
  scroll-snap-margin-top: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: -42365;
  -moz-scroll-snap-margin-top: -42365;
  -ms-scroll-snap-margin-top: -42365;
  scroll-snap-margin-top: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: -42365;
  -moz-scroll-snap-margin-top: -42365;
  -ms-scroll-snap-margin-top: -42365;
  scroll-snap-margin-top: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginTop renders object [breakpoints + standard] 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-top: July;
  -moz-scroll-snap-margin-top: July;
  -ms-scroll-snap-margin-top: July;
  scroll-snap-margin-top: July;
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: August;
    -moz-scroll-snap-margin-top: August;
    -ms-scroll-snap-margin-top: August;
    scroll-snap-margin-top: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: September;
    -moz-scroll-snap-margin-top: September;
    -ms-scroll-snap-margin-top: September;
    scroll-snap-margin-top: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: November;
    -moz-scroll-snap-margin-top: November;
    -ms-scroll-snap-margin-top: November;
    scroll-snap-margin-top: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-scroll-snap-margin-top: August;
    -moz-scroll-snap-margin-top: August;
    -ms-scroll-snap-margin-top: August;
    scroll-snap-margin-top: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: July;
  -moz-scroll-snap-margin-top: July;
  -ms-scroll-snap-margin-top: July;
  scroll-snap-margin-top: July;
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: August;
    -moz-scroll-snap-margin-top: August;
    -ms-scroll-snap-margin-top: August;
    scroll-snap-margin-top: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: September;
    -moz-scroll-snap-margin-top: September;
    -ms-scroll-snap-margin-top: September;
    scroll-snap-margin-top: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: November;
    -moz-scroll-snap-margin-top: November;
    -ms-scroll-snap-margin-top: November;
    scroll-snap-margin-top: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-scroll-snap-margin-top: August;
    -moz-scroll-snap-margin-top: August;
    -ms-scroll-snap-margin-top: August;
    scroll-snap-margin-top: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: January;
  -moz-scroll-snap-margin-top: January;
  -ms-scroll-snap-margin-top: January;
  scroll-snap-margin-top: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: December;
    -moz-scroll-snap-margin-top: December;
    -ms-scroll-snap-margin-top: December;
    scroll-snap-margin-top: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: April;
    -moz-scroll-snap-margin-top: April;
    -ms-scroll-snap-margin-top: April;
    scroll-snap-margin-top: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: May;
    -moz-scroll-snap-margin-top: May;
    -ms-scroll-snap-margin-top: May;
    scroll-snap-margin-top: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-scroll-snap-margin-top: June;
    -moz-scroll-snap-margin-top: June;
    -ms-scroll-snap-margin-top: June;
    scroll-snap-margin-top: June;
  }
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: January;
  -moz-scroll-snap-margin-top: January;
  -ms-scroll-snap-margin-top: January;
  scroll-snap-margin-top: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: December;
    -moz-scroll-snap-margin-top: December;
    -ms-scroll-snap-margin-top: December;
    scroll-snap-margin-top: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: April;
    -moz-scroll-snap-margin-top: April;
    -ms-scroll-snap-margin-top: April;
    scroll-snap-margin-top: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: May;
    -moz-scroll-snap-margin-top: May;
    -ms-scroll-snap-margin-top: May;
    scroll-snap-margin-top: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-scroll-snap-margin-top: June;
    -moz-scroll-snap-margin-top: June;
    -ms-scroll-snap-margin-top: June;
    scroll-snap-margin-top: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginTop renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: July;
    -moz-scroll-snap-margin-top: July;
    -ms-scroll-snap-margin-top: July;
    scroll-snap-margin-top: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: August;
    -moz-scroll-snap-margin-top: August;
    -ms-scroll-snap-margin-top: August;
    scroll-snap-margin-top: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: September;
    -moz-scroll-snap-margin-top: September;
    -ms-scroll-snap-margin-top: September;
    scroll-snap-margin-top: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-scroll-snap-margin-top: November;
    -moz-scroll-snap-margin-top: November;
    -ms-scroll-snap-margin-top: November;
    scroll-snap-margin-top: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: July;
    -moz-scroll-snap-margin-top: July;
    -ms-scroll-snap-margin-top: July;
    scroll-snap-margin-top: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: August;
    -moz-scroll-snap-margin-top: August;
    -ms-scroll-snap-margin-top: August;
    scroll-snap-margin-top: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: September;
    -moz-scroll-snap-margin-top: September;
    -ms-scroll-snap-margin-top: September;
    scroll-snap-margin-top: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-scroll-snap-margin-top: November;
    -moz-scroll-snap-margin-top: November;
    -ms-scroll-snap-margin-top: November;
    scroll-snap-margin-top: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: May;
    -moz-scroll-snap-margin-top: May;
    -ms-scroll-snap-margin-top: May;
    scroll-snap-margin-top: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: June;
    -moz-scroll-snap-margin-top: June;
    -ms-scroll-snap-margin-top: June;
    scroll-snap-margin-top: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: April;
    -moz-scroll-snap-margin-top: April;
    -ms-scroll-snap-margin-top: April;
    scroll-snap-margin-top: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-scroll-snap-margin-top: November;
    -moz-scroll-snap-margin-top: November;
    -ms-scroll-snap-margin-top: November;
    scroll-snap-margin-top: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: May;
    -moz-scroll-snap-margin-top: May;
    -ms-scroll-snap-margin-top: May;
    scroll-snap-margin-top: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: June;
    -moz-scroll-snap-margin-top: June;
    -ms-scroll-snap-margin-top: June;
    scroll-snap-margin-top: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-top: April;
    -moz-scroll-snap-margin-top: April;
    -ms-scroll-snap-margin-top: April;
    scroll-snap-margin-top: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-scroll-snap-margin-top: November;
    -moz-scroll-snap-margin-top: November;
    -ms-scroll-snap-margin-top: November;
    scroll-snap-margin-top: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginTop renders object [mediaQueries + standart] 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-top: July;
  -moz-scroll-snap-margin-top: July;
  -ms-scroll-snap-margin-top: July;
  scroll-snap-margin-top: July;
}

@media May {
  .c0 {
    -webkit-scroll-snap-margin-top: August;
    -moz-scroll-snap-margin-top: August;
    -ms-scroll-snap-margin-top: August;
    scroll-snap-margin-top: August;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: September;
    -moz-scroll-snap-margin-top: September;
    -ms-scroll-snap-margin-top: September;
    scroll-snap-margin-top: September;
  }
}

@media January {
  .c0 {
    -webkit-scroll-snap-margin-top: November;
    -moz-scroll-snap-margin-top: November;
    -ms-scroll-snap-margin-top: November;
    scroll-snap-margin-top: November;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: August;
    -moz-scroll-snap-margin-top: August;
    -ms-scroll-snap-margin-top: August;
    scroll-snap-margin-top: August;
  }
}

@media June {
  .c0 {
    -webkit-scroll-snap-margin-top: November;
    -moz-scroll-snap-margin-top: November;
    -ms-scroll-snap-margin-top: November;
    scroll-snap-margin-top: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: July;
  -moz-scroll-snap-margin-top: July;
  -ms-scroll-snap-margin-top: July;
  scroll-snap-margin-top: July;
}

@media May {
  .c0 {
    -webkit-scroll-snap-margin-top: August;
    -moz-scroll-snap-margin-top: August;
    -ms-scroll-snap-margin-top: August;
    scroll-snap-margin-top: August;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: September;
    -moz-scroll-snap-margin-top: September;
    -ms-scroll-snap-margin-top: September;
    scroll-snap-margin-top: September;
  }
}

@media January {
  .c0 {
    -webkit-scroll-snap-margin-top: November;
    -moz-scroll-snap-margin-top: November;
    -ms-scroll-snap-margin-top: November;
    scroll-snap-margin-top: November;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: August;
    -moz-scroll-snap-margin-top: August;
    -ms-scroll-snap-margin-top: August;
    scroll-snap-margin-top: August;
  }
}

@media June {
  .c0 {
    -webkit-scroll-snap-margin-top: November;
    -moz-scroll-snap-margin-top: November;
    -ms-scroll-snap-margin-top: November;
    scroll-snap-margin-top: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: April;
  -moz-scroll-snap-margin-top: April;
  -ms-scroll-snap-margin-top: April;
  scroll-snap-margin-top: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    -webkit-scroll-snap-margin-top: November;
    -moz-scroll-snap-margin-top: November;
    -ms-scroll-snap-margin-top: November;
    scroll-snap-margin-top: November;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: January;
    -moz-scroll-snap-margin-top: January;
    -ms-scroll-snap-margin-top: January;
    scroll-snap-margin-top: January;
  }
}

@media January {
  .c0 {
    -webkit-scroll-snap-margin-top: December;
    -moz-scroll-snap-margin-top: December;
    -ms-scroll-snap-margin-top: December;
    scroll-snap-margin-top: December;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: April;
    -moz-scroll-snap-margin-top: April;
    -ms-scroll-snap-margin-top: April;
    scroll-snap-margin-top: April;
  }
}

@media June {
  .c0 {
    -webkit-scroll-snap-margin-top: May;
    -moz-scroll-snap-margin-top: May;
    -ms-scroll-snap-margin-top: May;
    scroll-snap-margin-top: May;
  }
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: April;
  -moz-scroll-snap-margin-top: April;
  -ms-scroll-snap-margin-top: April;
  scroll-snap-margin-top: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    -webkit-scroll-snap-margin-top: November;
    -moz-scroll-snap-margin-top: November;
    -ms-scroll-snap-margin-top: November;
    scroll-snap-margin-top: November;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: January;
    -moz-scroll-snap-margin-top: January;
    -ms-scroll-snap-margin-top: January;
    scroll-snap-margin-top: January;
  }
}

@media January {
  .c0 {
    -webkit-scroll-snap-margin-top: December;
    -moz-scroll-snap-margin-top: December;
    -ms-scroll-snap-margin-top: December;
    scroll-snap-margin-top: December;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: April;
    -moz-scroll-snap-margin-top: April;
    -ms-scroll-snap-margin-top: April;
    scroll-snap-margin-top: April;
  }
}

@media June {
  .c0 {
    -webkit-scroll-snap-margin-top: May;
    -moz-scroll-snap-margin-top: May;
    -ms-scroll-snap-margin-top: May;
    scroll-snap-margin-top: May;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginTop renders object [mediaQueries] 1`] = `
Array [
  @media May {
  .c0 {
    -webkit-scroll-snap-margin-top: July;
    -moz-scroll-snap-margin-top: July;
    -ms-scroll-snap-margin-top: July;
    scroll-snap-margin-top: July;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: August;
    -moz-scroll-snap-margin-top: August;
    -ms-scroll-snap-margin-top: August;
    scroll-snap-margin-top: August;
  }
}

@media January {
  .c0 {
    -webkit-scroll-snap-margin-top: September;
    -moz-scroll-snap-margin-top: September;
    -ms-scroll-snap-margin-top: September;
    scroll-snap-margin-top: September;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: November;
    -moz-scroll-snap-margin-top: November;
    -ms-scroll-snap-margin-top: November;
    scroll-snap-margin-top: November;
  }
}

@media June {
  .c0 {
    -webkit-scroll-snap-margin-top: August;
    -moz-scroll-snap-margin-top: August;
    -ms-scroll-snap-margin-top: August;
    scroll-snap-margin-top: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    -webkit-scroll-snap-margin-top: July;
    -moz-scroll-snap-margin-top: July;
    -ms-scroll-snap-margin-top: July;
    scroll-snap-margin-top: July;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: August;
    -moz-scroll-snap-margin-top: August;
    -ms-scroll-snap-margin-top: August;
    scroll-snap-margin-top: August;
  }
}

@media January {
  .c0 {
    -webkit-scroll-snap-margin-top: September;
    -moz-scroll-snap-margin-top: September;
    -ms-scroll-snap-margin-top: September;
    scroll-snap-margin-top: September;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: November;
    -moz-scroll-snap-margin-top: November;
    -ms-scroll-snap-margin-top: November;
    scroll-snap-margin-top: November;
  }
}

@media June {
  .c0 {
    -webkit-scroll-snap-margin-top: August;
    -moz-scroll-snap-margin-top: August;
    -ms-scroll-snap-margin-top: August;
    scroll-snap-margin-top: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    -webkit-scroll-snap-margin-top: January;
    -moz-scroll-snap-margin-top: January;
    -ms-scroll-snap-margin-top: January;
    scroll-snap-margin-top: January;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: December;
    -moz-scroll-snap-margin-top: December;
    -ms-scroll-snap-margin-top: December;
    scroll-snap-margin-top: December;
  }
}

@media January {
  .c0 {
    -webkit-scroll-snap-margin-top: April;
    -moz-scroll-snap-margin-top: April;
    -ms-scroll-snap-margin-top: April;
    scroll-snap-margin-top: April;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: May;
    -moz-scroll-snap-margin-top: May;
    -ms-scroll-snap-margin-top: May;
    scroll-snap-margin-top: May;
  }
}

@media June {
  .c0 {
    -webkit-scroll-snap-margin-top: June;
    -moz-scroll-snap-margin-top: June;
    -ms-scroll-snap-margin-top: June;
    scroll-snap-margin-top: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    -webkit-scroll-snap-margin-top: January;
    -moz-scroll-snap-margin-top: January;
    -ms-scroll-snap-margin-top: January;
    scroll-snap-margin-top: January;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: December;
    -moz-scroll-snap-margin-top: December;
    -ms-scroll-snap-margin-top: December;
    scroll-snap-margin-top: December;
  }
}

@media January {
  .c0 {
    -webkit-scroll-snap-margin-top: April;
    -moz-scroll-snap-margin-top: April;
    -ms-scroll-snap-margin-top: April;
    scroll-snap-margin-top: April;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-top: May;
    -moz-scroll-snap-margin-top: May;
    -ms-scroll-snap-margin-top: May;
    scroll-snap-margin-top: May;
  }
}

@media June {
  .c0 {
    -webkit-scroll-snap-margin-top: June;
    -moz-scroll-snap-margin-top: June;
    -ms-scroll-snap-margin-top: June;
    scroll-snap-margin-top: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginTop renders postitive number 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-top: 54881;
  -moz-scroll-snap-margin-top: 54881;
  -ms-scroll-snap-margin-top: 54881;
  scroll-snap-margin-top: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: 54881;
  -moz-scroll-snap-margin-top: 54881;
  -ms-scroll-snap-margin-top: 54881;
  scroll-snap-margin-top: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: 42365;
  -moz-scroll-snap-margin-top: 42365;
  -ms-scroll-snap-margin-top: 42365;
  scroll-snap-margin-top: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: 42365;
  -moz-scroll-snap-margin-top: 42365;
  -ms-scroll-snap-margin-top: 42365;
  scroll-snap-margin-top: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginTop renders string 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-top: July;
  -moz-scroll-snap-margin-top: July;
  -ms-scroll-snap-margin-top: July;
  scroll-snap-margin-top: July;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: July;
  -moz-scroll-snap-margin-top: July;
  -ms-scroll-snap-margin-top: July;
  scroll-snap-margin-top: July;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: June;
  -moz-scroll-snap-margin-top: June;
  -ms-scroll-snap-margin-top: June;
  scroll-snap-margin-top: June;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: June;
  -moz-scroll-snap-margin-top: June;
  -ms-scroll-snap-margin-top: June;
  scroll-snap-margin-top: June;
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginTop renders undefined 1`] = `
Array [
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
]
`;

exports[`scrollSnapMarginTop renders valid theme path 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-top: DarkSlateBlue;
  -moz-scroll-snap-margin-top: DarkSlateBlue;
  -ms-scroll-snap-margin-top: DarkSlateBlue;
  scroll-snap-margin-top: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: DarkSlateBlue;
  -moz-scroll-snap-margin-top: DarkSlateBlue;
  -ms-scroll-snap-margin-top: DarkSlateBlue;
  scroll-snap-margin-top: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: DarkSlateBlue;
  -moz-scroll-snap-margin-top: DarkSlateBlue;
  -ms-scroll-snap-margin-top: DarkSlateBlue;
  scroll-snap-margin-top: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-top: DarkSlateBlue;
  -moz-scroll-snap-margin-top: DarkSlateBlue;
  -ms-scroll-snap-margin-top: DarkSlateBlue;
  scroll-snap-margin-top: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
